lib/marquee.dart

```diff
@@ -483,6 +484,9 @@ class Marquee extends StatefulWidget {
   /// * [accelerationCurve], the equivalent for decelerating.
   final _IntegralCurve decelerationCurve;
 
+  /// The function that will be performed when all rounds are complete.
+  final Function? onDone;
```

Comment:
You shouldn't be able to pass in a function that accepts a parameter.
```suggestion
  final VoidCallback? onDone;
```